django/contrib/gis/templates/gis/admin/openlayers.js

```diff
@@ -12,98 +12,98 @@ OpenLayers.Projection.addTransform("EPSG:4326", "EPSG:3857", OpenLayers.Layer.Sp
 {{ module }}.is_point = {{ is_point|yesno:"true,false" }};
 {% endblock %}
 {{ module }}.get_ewkt = function(feat){
-  return 'SRID={{ srid|unlocalize }};' + {{ module }}.wkt_f.write(feat);
+    return 'SRID={{ srid|unlocalize }};' + {{ module }}.wkt_f.write(feat);
 };
 {{ module }}.read_wkt = function(wkt){
-  // OpenLayers cannot handle EWKT -- we make sure to strip it out.
-  // EWKT is only exposed to OL if there's a validation error in the admin.
-  var match = {{ module }}.re.exec(wkt);
-  if (match){wkt = match[1];}
-  return {{ module }}.wkt_f.read(wkt);
+    // OpenLayers cannot handle EWKT -- we make sure to strip it out.
+    // EWKT is only exposed to OL if there's a validation error in the admin.
+    var match = {{ module }}.re.exec(wkt);
+    if (match){wkt = match[1];}
+    return {{ module }}.wkt_f.read(wkt);
 };
 {{ module }}.write_wkt = function(feat){
-  if ({{ module }}.is_collection){ {{ module }}.num_geom = feat.geometry.components.length;}
-  else { {{ module }}.num_geom = 1;}
-  document.getElementById('{{ id }}').value = {{ module }}.get_ewkt(feat);
+    if ({{ module }}.is_collection){ {{ module }}.num_geom = feat.geometry.components.length;}
+    else { {{ module }}.num_geom = 1;}
+    document.getElementById('{{ id }}').value = {{ module }}.get_ewkt(feat);
 };
 {{ module }}.add_wkt = function(event){
-  // This function will sync the contents of the `vector` layer with the
-  // WKT in the text field.
-  if ({{ module }}.is_collection){
-    var feat = new OpenLayers.Feature.Vector(new OpenLayers.Geometry.{{ geom_type }}());
-    for (var i = 0; i < {{ module }}.layers.vector.features.length; i++){
-      feat.geometry.addComponents([{{ module }}.layers.vector.features[i].geometry]);
-    }
-    {{ module }}.write_wkt(feat);
-  } else {
-    // Make sure to remove any previously added features.
-    if ({{ module }}.layers.vector.features.length > 1){
-      old_feats = [{{ module }}.layers.vector.features[0]];
-      {{ module }}.layers.vector.removeFeatures(old_feats);
-      {{ module }}.layers.vector.destroyFeatures(old_feats);
+    // This function will sync the contents of the `vector` layer with the
+    // WKT in the text field.
+    if ({{ module }}.is_collection){
```

Comment:
That may be worth it if its easy. FWIW when we switched, our developers on several different platforms and editors were all easily able to find ESLint plugins to replace their JSHint plugins. I know at least Sublime Text and Emacs have ESLint support; not sure which other ones.